src/drivers/MemoryDriver.ts

```diff
@@ -0,0 +1,57 @@
+import { IDriver } from "./IDriver";
+
+type Table = Map<string, any>;
+
+export class MemoryDriver implements IDriver {
+    public $store = new Map<string, Table>();
+
+    public async prepare(table: string): Promise<void> {
+        this.$getOrCreateTable(table);
+    }
+
+    public $getOrCreateTable(name: string) {
+        if (this.$store.has(name)) return this.$store.get(name)!;
```

Comment:
This could be done in one method call by only using get and if the value return from it is not null retuning it.

It will also remove the `Forbidden non-null assertion` warning from eslint